tests/integration/testlogger.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
Seems like this file is a duplicate with `tests/e2e/testlogger.go`, should be moved to a shared/common package.